src/Components/CreateImageWizardV2/steps/Oscap/Oscap.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useEffect, useState } from 'react';
+
+import {
+  Alert,
+  FormGroup,
+  Spinner,
+  Popover,
+  TextContent,
+  Text,
+  Button,
+} from '@patternfly/react-core';
+import {
+  Select,
+  SelectOption,
+  SelectVariant,
+} from '@patternfly/react-core/deprecated';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import OscapProfileInformation from './OscapProfileInformation';
+
+import { useAppDispatch, useAppSelector } from '../../../../store/hooks';
+import {
+  DistributionProfileItem,
+  useGetOscapCustomizationsQuery,
+  useGetOscapProfilesQuery,
+} from '../../../../store/imageBuilderApi';
+import {
+  changeOscapProfile,
+  selectDistribution,
+  selectProfile,
+} from '../../../../store/wizardSlice';
+
+/**
+ * Component for the user to select the profile to apply to their image.
+ * The selected profile will be stored in the `oscap-profile` form state variable.
+ * The Component is shown or not depending on the ShowSelector variable.
+ */
+
+const ProfileSelector = () => {
+  const oscapProfile = useAppSelector((state) => selectProfile(state));
+  const release = useAppSelector((state) => selectDistribution(state));
+  const dispatch = useAppDispatch();
+  const [profileName, setProfileName] = useState<string | undefined>('');
```

Comment:
Isn't the profileName and oscapProfile a duplicate value?

What do we plan to store in each of those?

Reply:
no, the oscapProfile contains the profile_ id  abd profileName contains the profile_name